modules/public/views.py

```diff
@@ -178,8 +178,12 @@ def snap_details(snap_name):
     status_code = 200
     country_data = []
     try:
-        today = datetime.datetime.utcnow().date()
-        yesterday = today - relativedelta.relativedelta(days=1)
+        # We want to give time to the store to preoccess all the metrics,
+        # since the metrics are processed during the night
+        # https://github.com/canonical-websites/snapcraft.io/pull/616
+        today = datetime.datetime.utcnow() - relativedelta.relativedelta(
+            hours=12)
+        yesterday = today.date() - relativedelta.relativedelta(days=1)
```

Comment:
I agree with @nottrobin previous comment, that it's quite confusing naming it `today` and `yesterday` if they are not really that.